### PR TITLE
OI-70 Personalized activity stream

### PR DESCRIPTION
**Pull request by nvelychenko**

### CODE REVIEW

File: modules/openideal_idea/templates/openideal-user-activity-list.html.twig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+/**
+* @file
+* Default theme implementation to display a block.
+*
+* @see template_preprocess_block()
+*
+* @ingroup themeable
+*/
+#}
+<div class="activity-stream">
```
Comment (db-Matroskeen) [inline modules/openideal_idea/templates/openideal-user-activity-list.html.twig]: Can't we use default item-list?
https://api.drupal.org/api/drupal/core%21modules%21system%21templates%21item-list.html.twig/8.4.x

File: themes/openideal_theme/openideal_theme.libraries.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ global-styling:
   css:
     component:
       '//fonts.googleapis.com/css2?family=Assistant:wght@300;400;600;700&family=Roboto&display=swap': { type: external, minified: true }
-      css/style.css: {}
+      css/style.min.css: {}
```
Comment (db-Matroskeen) [inline themes/openideal_theme/openideal_theme.libraries.yml]: Is that necessary?

File: modules/openideal_user/src/Plugin/Block/OpenidealUserPersonalActivityStream.php [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+  /**
+   * Fetch activity stream message ids.
+   *
+   * @Todo: Find the way to check permissions when the private flavours module is enabled.
+   * Because of "node_grants" were removed from the Group module.
+   *
+   * @return array|null
+   *   Fetched entities.
+   */
+  protected function fetchContent() {
```
Comment (db-Matroskeen) [inline modules/openideal_user/src/Plugin/Block/OpenidealUserPersonalActivityStream.php]: 1) Can we avoid duplicated code?
2) Can we avoid variables like `second_query`, `third_query`?

**Review by db-Matroskeen — CHANGES_REQUESTED** [review]

File: modules/openideal_user/src/Plugin/Block/OpenidealUserPersonalActivityStream.php (hunk 3 above)
Comment (nvelychenko) [reply modules/openideal_user/src/Plugin/Block/OpenidealUserPersonalActivityStream.php]: 2. Done.

File: themes/openideal_theme/openideal_theme.libraries.yml (hunk 2 above)
Comment (nvelychenko) [reply themes/openideal_theme/openideal_theme.libraries.yml]: Yes.

File: modules/openideal_idea/templates/openideal-user-activity-list.html.twig (hunk 1 above)
Comment (nvelychenko) [reply modules/openideal_idea/templates/openideal-user-activity-list.html.twig]: Yes, we can. Used.

**Pull request merged**